CMakeVOL.cmake

```diff
@@ -39,6 +39,13 @@ if (HDF5_VOL_ALLOW_EXTERNAL)
   set (HDF5_FOUND 1)
   set (HDF5_LIBRARIES "${HDF5_LIBSH_TARGET};${LINK_LIBS};${LINK_COMP_LIBS};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:MPI::MPI_C>")
   set (HDF5_INCLUDE_DIRS "${HDF5_SRC_INCLUDE_DIRS};${HDF5_SRC_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_INCLUDE_DIRS}>")
+  set (HDF5_DIR "${HDF5_SOURCE_DIR}")
+
+  set (HDF5_C_LIBRARIES "${HDF5_LIB_TARGETS}")
```

Comment:
Hmm, shouldn't this essentially be the same as `HDF5_LIBRARIES`?

Reply:
Fixed - this was initially to prevent spurious dependency creation in the REST VOL CMake, but I fixed that on the REST VOL's side.